core/theme/zelos.js

```diff
@@ -101,6 +102,8 @@ Blockly.Themes.Zelos.categoryStyles = {
   }
 };
 
-Blockly.Themes.Zelos =
-    new Blockly.Theme('zelos', Blockly.Themes.Zelos.defaultBlockStyles,
-        Blockly.Themes.Zelos.categoryStyles);
+Zelos =
+    new Blockly.Theme('zelos', Zelos.defaultBlockStyles,
+        Zelos.categoryStyles);
```

Comment:
Does reassigning this clobber the `defaultBlockStyles` and `categoryStyles` properties?

Reply:
No. To be sure I've also tested with the advanced playground and I can see that the `defaultBlockStyles` and `categoryStyles` are preserved correctly.